src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -106,22 +111,22 @@ public void autonomousInit() {
 
 		// TODO: call appropriate auto modes based on list
 		LOGGER.debug("Autonomous init: " + autoMode);
-		switch (autoMode) {
-		case "none":
+//		switch (autoMode) {
+//		case "none":
 //			autonomous = Actions.doNothing();
-			break;
-		default:
+//			break;
+//		default:
 //			autonomous = Actions.doNothing();
-			break;
-		}
-		LOGGER.info("Init Autonomous:" + autonomous.getName());
-		autonomous.enable();
+//			break;
+//		}
+//		LOGGER.info("Init Autonomous:" + autonomous.getName());
+//		autonomous.enable();
```

Comment:
This should be commented as well. Basically, if you comment out code and leave it you need to either comment it or remove it.